### PR TITLE
test(docs): add script to test markdown documentation

### PR DESCRIPTION
**Pull request by platten**
Signed-off-by: Paul Pietkiewicz <paul@profian.com>

### CODE REVIEW

File: helper/md-test.sh [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+readonly HOMEDIR="/home/user"
+
+test_image_kvm() {
+    local markdown_doc_path="$(realpath "$1")"
+    local image="$2"
+    local context="$3"
+    echo -e "\n\nRunning: ${image} (KVM)"
+    echo "Context: \"${context}\""
+    echo "Markdown Document: \"${markdown_doc_path}\""
+    docker run \
```
Comment (haraldh) [inline helper/md-test.sh:23]: why not use podman if you run on our self-runners anyway?

**Comment by platten** [comment]
Blocked by #1795

File: helper/md-test.sh (hunk 1 above)
Comment (platten) [reply helper/md-test.sh]: We are not running on our self-hosted runners. KVM is for local testing. Goal is to transition to the direct/nil backend and remove the KVM backend completely.

**Comment by platten** [comment]
- [Example passed job](https://github.com/platten/enarx/runs/6214463903?check_suite_focus=true)
- [Example failed job](https://github.com/platten/enarx/actions/runs/2240432332)
- [Example issue filed](https://github.com/platten/enarx/issues/114)

**Pull request merged**